packages/@uppy/core/tsconfig.build.json

```diff
@@ -0,0 +1,20 @@
+{
+  "extends": "../../../tsconfig.shared",
+  "compilerOptions": {
+    "outDir": "./lib",
+    "rootDir": "./src",
+    "resolveJsonModule": false,
+    "noImplicitAny": false,
+    "skipLibCheck": true
```

Comment:
I guess that's what causing production TS to ignore untyped dependencies, and because it's not present in `tsconfig.json`, it reports an error. Maybe we want to re-enable it on prod (and have `@ts-expect-error` in source files), or we'd want to skip it on the shared options